components/desktop/gnome2/gnome-media/Makefile

```diff
@@ -62,6 +63,3 @@ install:	$(INSTALL_32)
 
 test:		$(NO_TESTS)
 
-BUILD_PKG_DEPENDENCIES = $(BUILD_TOOLS)
-
-include $(WS_TOP)/make-rules/depend.mk
```

Comment:
you can start from gmake REQUIRED_PACKAGES and look at configure checks for missing dependencies....